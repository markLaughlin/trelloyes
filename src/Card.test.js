import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';

describe('My component', () => {
    //all my test cases here
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Card />, div);
        ReactDOM.unmountComponentAtNode(div);
      });

      it('renders correctly; snapshot test', () => {
        const tree = renderer
          .create(<Card/>)
          .toJSON();
        expect(tree).toMatchSnapshot();
      });
   });