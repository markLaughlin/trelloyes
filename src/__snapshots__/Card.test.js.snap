// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My component renders correctly; snapshot test 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;
